winui/Rating/Clear-rating.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: Clear rating value in WinUI Rating control | Syncfusion
+description: Learn all about how to clear the rating value in the WinUI Rating (SfRating) control.
+platform: WinUI
+control: SfRating
+documentation: ug
+---
+
+# Clear rating value in WinUI Rating
+
+The `IsClearEnabled` property of the Rating control allows you to clear the provided rating value when the value is true. The default value of the `IsClearEnabled` property is `true`.
```

Comment:
The `IsClearEnabled` property of the Rating control allows you to clear the provided rating value when the value is true. The default value of the `IsClearEnabled` property is `true.`

Reply:
Updated the above mentioned changes.